forms-flow-web/src/components/Modeler/SubFlowTable.js

```diff
@@ -44,19 +44,31 @@ const SubFlow = React.memo(() => {
           processType: "BPMN",
           limit: limit,
           searchKey: search,
-          sortBy: Object.keys(sort),
-          sortOrder: Object.values(sort),
+          sortBy: currentBPMNsort.sortBy,
```

Comment:
here we are accept only one sorting at a time or multiple sorting ?

Reply:
One sorting at a time.

Reply:
but in ui we can sort multiple things right ? @arun-s-aot